docs/website/docs/building-from-source/optional-features.md

```diff
@@ -86,7 +89,11 @@ Also see [instructions for installing pre-built binaries](../../bindings/python/
     === "Linux and MacOS"
 
         ``` shell
-        cmake -DIREE_BUILD_PYTHON_BINDINGS=ON -DPython3_EXECUTABLE="$(which python)" .
+        cmake \
+          -DIREE_BUILD_PYTHON_BINDINGS=ON \
+          -DPython3_EXECUTABLE="$(which python)" \
```

Comment:
`which python3` might be better here? (On Ubuntu you can have no default `python` nowadays but `python3` should exist.)

Reply:
See above:

```
# Make sure your 'python' is what you expect. Note that on multi-python
# systems, this may have a version suffix, and on many Linuxes and MacOS where
# python2 and python3 co-exist, you may also want to use `python3`.
which python
python --version
```